src/test/functional/steps/local-authorities.ts

```diff
@@ -65,24 +77,29 @@ Then('An error is displayed for local authorities with title {string} and summer
 When('I will make sure Family court type is not selected', async () => {
   const selectorFamilyCourt = '#court_types-2';
   const selectorTribunalCourt = '#court_types-3';
-
-  const elementFamilyCourtExist = await I.checkElement(selectorFamilyCourt);
-  expect(elementFamilyCourtExist).equal(true);
-  const elementTribunalCourtExist = await I.checkElement(selectorTribunalCourt);
-  expect(elementTribunalCourtExist).equal(true);
-
-  const elementFamilyCourtChecked = await I.isElementChecked(selectorFamilyCourt);
-  if (elementFamilyCourtChecked) {
-    await I.click(selectorFamilyCourt);
-  }
-  const elementTribunalCourtChecked = await I.isElementChecked(selectorTribunalCourt);
-  if (!elementTribunalCourtChecked) {
-    await I.click(selectorTribunalCourt);
+  const elementVisible = await I.isElementVisible(selectorFamilyCourt);
+
+  if (elementVisible) {
```

Comment:
I'm not sure why this change has been made - could you explain? If the element isn't visible, the 'checkElement' call won't be made so the step won't fail.

Reply:
I was getting a issue that the check for the element was happen before the page fully loaded so the test kept falling. So I put in a extra check for visibility which makes sure the element  is loaded fully

Reply:
I tried running the test in debug mode and the elementVisible was 'false' so the code in the block wasn't run.

Reply:
I have removed the elementVisible variable and fixed the issue